docs/docs/fallback-handoff.mdx

```diff
@@ -2,176 +2,148 @@
 id: fallback-handoff
 sidebar_label: Fallback and Human Handoff
 title: Fallback and Human Handoff
+abstract: When conversational assistants fail, it's important that they do so gracefully.
 ---
 
 import useBaseUrl from '@docusaurus/useBaseUrl';
 
-<!-- TODO: add info about human handoff -->
-
-<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="failing-gracefully"></a>
 
 Even if you design your bot perfectly, users will inevitably say things to your
 assistant that you did not anticipate. In these cases, your assistant will fail,
 and it's important you ensure it does so gracefully.
 
-## Fallback policy
-
-One of the most common failures is low NLU confidence, which is handled very nicely with
-the TwoStageFallbackPolicy. You can enable it by adding the following to your configuration file,
+## Handling Out-of-Scope Messages
 
-```yaml
-policies:
-  - name: TwoStageFallbackPolicy
-    nlu_threshold: 0.8
-```
+It is good practice to handle questions you know your users may ask,
+but for which you haven't implemented a user goal yet.
 
-and adding the `out_of_scope` intent to your `domain.yml` file:
+You can define an `out_of_scope` intent to handle generic out of scope requests, like “I'm hungry” and have
+the bot respond with a default message like “Sorry, I can't handle that request” by writing a rule with the
+`out_of_scope` intent in it.
 
-```yaml
-intents:
-- out_of_scope
-```
 
-When the nlu confidence falls below the defined threshold, the bot will prompt the user to
-rephrase their message. If the bot isn't able to get their message three times, there
-will be a final action where the bot can e.g. hand off to a human.
+### 1. Creating an Out-of-Scope intent
 
-To try this out, retrain your model and send a message like “order me a pizza” to your bot:
+You will first need to define an `out_of_scope` intent in your NLU training data and add any known
+out-of-scope requests as training examples, for example:
 
-```bash
-rasa train
-rasa shell
+```yaml
+nlu:
+- intent: out_of_scope
+  examples: |
+    - I want to order food
+    - What is 2 + 2?
+    - Who's the US President?
 ```
 
-There are also a bunch of ways in which you can customize this policy. In Sara, our demo bot,
-we've customized it to suggest intents to the user within a certain confidence range to make
-it easier for the user to give the bot the information it needs.
+As with every intent, you should source the majority of your examples
+from real conversations. This is especially important for out-of-scope intents
+since false positives for these intents can lead to a bad user experience.
```

Comment:
This is what I meant, but I'm taking it out as too pedantic
```
This is especially important for out-of-scope intents.
If the training data for the out-of-scope intent is too broad or not representative,
in-scope user messages can be treated as out-of-scope.
This can lead to a bad user experience since users get annoyed by repeated "I can't help you" messages.
```